sequence_analysis/src/cpp/SConscript

```diff
@@ -36,6 +36,9 @@ lib_env.AppendUnique(LIBS=['vptool', 'vpstat_tool'])
 lib_env.AppendUnique(CPPDEFINES=['MESSAGE'])
 inc = lib_env.ALEAIncludes( "sequence_analysis", includes )
 
+if env['debug']:
+    env.Append(CPPDEFINES= ['DEBUG'])
```

Comment:
Christophe: maybe there is a best way to do this.
If you validate this change you can propagate it everywhere in StructureAnalysis